modules/objdetect/src/aruco/charuco_detector.cpp

```diff
@@ -509,6 +509,7 @@ void drawDetectedCornersCharuco(InputOutputArray _image, InputArray _charucoCorn
               (_image.getMat().channels() == 1 || _image.getMat().channels() == 3));
     CV_Assert((_charucoCorners.getMat().total() == _charucoIds.getMat().total()) ||
               _charucoIds.getMat().total() == 0);
+    CV_Assert(_charucoCorners.getMat().type() == CV_32FC2);
 
     size_t nCorners = _charucoCorners.getMat().total();
```

Comment:
InputArray has  `total()` method. getMat() is redundant.

Reply:
removed getMat()